src/controllers/change.answers.controller.ts

```diff
@@ -0,0 +1,19 @@
+import {NextFunction, Request, Response} from "express";
+import {addToObjectionSession} from "../services/objection.session.service";
+import {Session} from "ch-node-session-handler";
+import {STRIKE_OFF_OBJECTIONS} from "../model/page.urls";
+import {CHANGE_ANSWER_KEY} from "../constants";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  const session: Session | undefined = req.session as Session;
+  if (session) {
+    const query: string | undefined = req.query["changePage"] as string;
+    if (query) {
+      addToObjectionSession(session, CHANGE_ANSWER_KEY, true);
```

Comment:
Sorry Matt, that comment was supposed to go here, but yes, does the scope of this story include clearing this flag on the get check your answers controller

Reply:
OBJ-287 was created for this.

Reply:
Yep, I agree with you